FIX #53528: l10n_ve_payment_extension

rsgg04 opened the pull request:
.-Se mejora flujo correspondiente del registro y aprobación de las retenciones.

Tarea (Link):
https://binaural.odoo.com/web\#id\=53528\&cids\=2\&menu_id\=975\&action\=341\&model\=project.task\&view_type\=form

Tarea de proyecto [x]
Ticket de soporte []

rsgg04 committed:
FIX #53528: l10n_ve_payment_extension

.-Se mejora flujo correspondiente del registro y aprobación de las retenciones.

Tarea (Link):
https://binaural.odoo.com/web\#id\=53528\&cids\=2\&menu_id\=975\&action\=341\&model\=project.task\&view_type\=form

Tarea de proyecto [x]
Ticket de soporte []

rsgg04 committed:
Merge branch '17.0' of github.com:binaural-dev/odoo-venezuela into 17.0_FEEAT-ta_53528_update_payment_extension

rsgg04 committed:
FIX #53528: l10n_ve_payment_extension

.-Se elimina validación para el numero de comprobante para los ISLR.

Tarea (Link):
https://binaural.odoo.com/web\#id\=53528\&cids\=2\&menu_id\=975\&action\=341\&model\=project.task\&view_type\=form

Tarea de proyecto [x]
Ticket de soporte []

rsgg04 committed:
FIX #53528: l10n_ve_payment_extension

.-Se elimina unit_test para el numero de comprobante para los ISLR.

Tarea (Link):
https://binaural.odoo.com/web\#id\=53528\&cids\=2\&menu_id\=975\&action\=341\&model\=project.task\&view_type\=form

Tarea de proyecto [x]
Ticket de soporte []

rsgg04 committed:
FIX #53528: l10n_ve_payment_extension

.-Se corrige unit_test para el numero de comprobante de IVA.

Tarea (Link):
https://binaural.odoo.com/web\#id\=53528\&cids\=2\&menu_id\=975\&action\=341\&model\=project.task\&view_type\=form

Tarea de proyecto [x]
Ticket de soporte []

Carlos-Lin-Binaural commented on l10n_ve_payment_extension/models/account_retention.py:
```diff
@@ -803,15 +798,8 @@ def _reconcile_supplier_payment(self, payment):
-                raise UserError("No se puede hacer una retencion con este concepto de pago")
-            
-        
+                lambda l: l.account_id.account_type == "liability_payable" and l.debit > 0)[0]
```
y en caso de que no traiga nada? da error con el [0]

rsgg04 committed:
FIX #53528: l10n_ve_payment_extension

.-Fue agregada la validación sugerida con su respectiva traducción.

Tarea (Link):
https://binaural.odoo.com/web\#id\=53528\&cids\=2\&menu_id\=975\&action\=341\&model\=project.task\&view_type\=form

Tarea de proyecto [x]
Ticket de soporte []
